nominal/core/client.py

```diff
@@ -963,6 +965,36 @@ def get_data_review(self, rid: str) -> DataReview:
         response = self._clients.datareview.get(self._clients.auth_header, rid)
         return DataReview._from_conjure(self._clients, response)
 
+    def create_event(
+        self,
+        name: str,
+        asset_rids: Sequence[str],
+        start: datetime | IntegralNanosecondsUTC,
+        duration: timedelta,
```

Comment:
I wonder if it's more or less useful to default this to a duration of 0? that's kinda what the front-end does